redfish-core/lib/log_services.hpp

```diff
@@ -3407,4 +3470,965 @@ inline void requestRoutesPostCodesEntry(App& app)
             });
 }
 
+/**
+ * @brief API Used to add the supported HardwareIsolation LogServices Members
+ *
+ * @param[in] req - The HardwareIsolation redfish request (unused now).
+ * @param[in] asyncResp - The redfish response to return.
+ *
+ * @return The redfish response in the given buffer.
+ */
+inline void getSystemHardwareIsolationLogService(
+    const crow::Request& /* req */,
+    const std::shared_ptr<bmcweb::AsyncResp>& asyncResp)
+{
+    asyncResp->res.jsonValue["@odata.id"] =
```

Comment:
You can still get here even when you don't find any 

                    std::array<const char*, 1>{
                        "xyz.openbmc_project.HardwareIsolation.Create"});

Reply:
Checking backend support and if not supported then returning "404" error.